devcluster/config.py

```diff
@@ -69,7 +77,7 @@ class StageConfig(metaclass=abc.ABCMeta):
 
     @staticmethod
     def read(config: typing.Any, temp_dir: str) -> "StageConfig":
-        allowed = {"db", "master", "agent", "custom", "custom_docker"}
+        allowed = {"db", "elastic", "master", "agent", "custom", "custom_docker"}
```

Comment:
idk how the ordering works in devcluster, but you may want elastic first. i imagine it's the thing you wanna bring down least?

Reply:
Ok, I can adjust that.  Only the experiment config really matters but I generally like having things ordered consistently when it's not too much work.

Reply:
i just meant like, the state machine or devcluster. elastic is the slowest for sure.

Reply:
lol, edited my comment

Reply:
expconf nightmares